package.json

```diff
@@ -10,7 +10,7 @@
   "license": "MIT",
   "repository": "jshttp/http-errors",
   "dependencies": {
-    "depd": "~1.1.2",
+    "depd": "^2.0.0",
```

Comment:
Modules outside of the organization ideally should be pinned per our policy, at most the patch floated, but may as well just pin this one.

Reply:
bumped major & new history entry

Reply:
Thanks. Sorry if there was confusion around the version; the change is done at release time by npm version command. You don't actually include that change in the PR directly.